boltdesignsystem/bolt
# [6.x] DS-43: Video support to background component

## Jira

https://pegadigitalit.atlassian.net/browse/DS-43

## Summary

Support for background videos is added to the Background component.

## Details

The background template was updated to support videos without the need for custom markup and styles.

1. a new prop was added to the schema
2. background.twig was updated
3. background.scss was updated
4. a new doc page for background video was created
5. careers homepage on Academy was updated so that there's no need to add custom markup and styles
6. test page was created (`/pattern-lab/?p=tests-background-video`)
    - video with a form in the layout, with theme class usage
    - video with forms, light and dark mode usage
    - video with the forms, theme classes usage
    - video with text on it, theme classes usage

## How to test

Pull the branch. Check the test page with video examples and the new doc page. Confirm that video support works.

## Release notes

There's support for using videos with the background component in combination with bands and bolt-layout
